samples/snippets/src/test/java/com/example/spanner/GetCommitStatsSampleIT.java

```diff
@@ -0,0 +1,150 @@
+ * limitations under the License.
+ */
+
+package com.example.spanner;
+
+import static com.google.common.truth.Truth.assertThat;
+
+import com.google.cloud.spanner.DatabaseAdminClient;
+import com.google.cloud.spanner.DatabaseClient;
+import com.google.cloud.spanner.DatabaseId;
+import com.google.cloud.spanner.Instance;
+import com.google.cloud.spanner.KeySet;
+import com.google.cloud.spanner.Mutation;
+import com.google.cloud.spanner.Spanner;
+import com.google.cloud.spanner.SpannerOptions;
+import com.google.common.collect.ImmutableList;
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Iterator;
+import java.util.UUID;
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/**
+ * Integration tests for {@link GetCommitStatsSample}
+ */
+@RunWith(JUnit4.class)
+public class GetCommitStatsSampleIT {
+
+  private static String instanceId = System.getProperty("spanner.test.instance");
+  private static String databaseId = formatForTest(
+      System.getProperty("spanner.sample.database", "commitstatssample"));
+  private static DatabaseId dbId;
+  private static DatabaseAdminClient dbClient;
+  private static Spanner spanner;
+
+  @BeforeClass
+  public static void createTestDatabase() throws Exception {
+    final SpannerOptions options =
+        SpannerOptions.newBuilder().setAutoThrottleAdministrativeRequests().build();
+    spanner = options.getService();
+    dbClient = spanner.getDatabaseAdminClient();
+    if (instanceId == null) {
+      Iterator<Instance> iterator =
+          spanner.getInstanceAdminClient().listInstances().iterateAll().iterator();
+      if (iterator.hasNext()) {
+        instanceId = iterator.next().getId().getInstance();
+      }
+    }
+    dbId = DatabaseId.of(options.getProjectId(), instanceId, databaseId);
+    dbClient.dropDatabase(dbId.getInstanceId().getInstance(), dbId.getDatabase());
+    dbClient
+        .createDatabase(
+            instanceId,
+            databaseId,
+            ImmutableList.of(
+                "CREATE TABLE Singers ("
+                    + "  SingerId   INT64 NOT NULL,"
+                    + "  FirstName  STRING(1024),"
+                    + "  LastName   STRING(1024),"
+                    + "  SingerInfo BYTES(MAX)"
+                    + ") PRIMARY KEY (SingerId)",
+                "CREATE TABLE Albums ("
+                    + "  SingerId        INT64 NOT NULL,"
+                    + "  AlbumId         INT64 NOT NULL,"
+                    + "  AlbumTitle      STRING(MAX),"
+                    + "  MarketingBudget INT64"
+                    + ") PRIMARY KEY (SingerId, AlbumId),"
+                    + "  INTERLEAVE IN PARENT Singers ON DELETE CASCADE",
+                "CREATE INDEX AlbumsByAlbumTitle ON Albums(AlbumTitle)"))
```

Comment:
We don't need this index for this sample

Reply:
Removed